src/components/card-link/card-link.module.css

```diff
@@ -4,6 +4,7 @@ style applied to anchor elements.
 */
 .root {
   color: black;
+  height: 100%;
```

Comment:
🙌 I was too slow to review this in time, but kudos for going for the root cause here 💯